messages/password.generate.json

```diff
@@ -1,15 +1,18 @@
 {
-  "description": "generate a password for scratch org users\nGenerates and sets a random password for one or more scratch org users. Targets the usernames listed with the --onbehalfof parameter or the --targetusername parameter. Defaults to the defaultusername.\n\nIf you haven’t set a default Dev Hub, or if your scratch org isn’t associated with your default Dev Hub, --targetdevhubusername is required.\n\nTo see a password that was previously generated, run \"sfdx force:user:display\".",
+  "description": "generate a password for scratch org users\nGenerates and sets a random password for one or more scratch org users. Targets the usernames listed with the --onbehalfof parameter or the --targetusername parameter. Defaults to the defaultusername.\n\nIf you haven’t set a default Dev Hub, or if your scratch org isn’t associated with your default Dev Hub, --targetdevhubusername is require.\n\nTo change the password strength, set the --complexity parameter to a value between 0 and 5. \nEach value specifies the types of characters used in the generated password: \n\n0 - lower case letters only.\n1 - lower case letters and numbers only.\n2 - lower case letters and symbols only.\n3 - lower and upper case letters and numbers only\n4 - lower and upper case letters and symbols only\n5 - lower and upper case letters and numbers and symbols only \n\nTo see a password that was previously generated, run \"sfdx force:user:display\".",
   "examples": [
     "sfdx force:user:password:generate",
+    "sfdx force:user:password:generate -l 12",
+    "sfdx force:user:password:generate -c 3",
     "sfdx force:user:password:generate -u me@my.org --json",
     "sfdx force:user:password:generate -o \"user1@my.org,user2@my.org,user3@my.org\""
   ],
   "flags": {
-    "onBehalfOf": "comma-separated list of usernames or aliases to assign the password to"
+    "onBehalfOf": "comma-separated list of usernames or aliases to assign the password to",
+    "length": "[default: 13] number of characters in the generated password; valid values are between 8 and 20",
```

Comment:
I don't think you hard-code the default value (i.e. the part that says "`[default: 13]`" in the message.  I _think_ there's a programmatic way of doing this.  @mshanemc , am I right? Can you help @vamsimundra if so?

Reply:
And now that I think about it, is there a programmatic way to show the range of valid values in the --help, without adding it to the message?  Otherwise, the text looks good.

Reply:
> I don't think you hard-code the default value (i.e. the part that says "[default: 13]" in the message. 

that's right!  If you put the default in the flag, the help output will automatically add it for you!

I don't remember if min/max show up automatically or not; you'd have to test that.